backend/src/modules/organization/exceptions/exceptions.ts

```diff
@@ -0,0 +1,13 @@
+export enum OrganizationExceptionCodes {
+  ORG_001 = 'ORG_001',
+  ORG_002 = 'ORG_002',
+}
+
+// Add messages for the coresponding Internal Exception Code
+export const OrganizationExceptionMessages: Record<
+  OrganizationExceptionCodes,
+  string
```

Comment:
cred ca in loc de `string` aici ar trebui sa intoarcem exact modelul `message, code` ca atunci cand facem throw sa facem `OrganizationExceptions.ORG_001`, sa nu mai construim din doua obiecte